src/Compilers/Core/MSBuildTask/Microsoft.Managed.Core.targets

```diff
@@ -4,7 +4,7 @@
   <!--
     Common targets for managed compilers.
   -->
-  <UsingTask TaskName="Microsoft.CodeAnalysis.BuildTasks.MapSourceRoots" AssemblyFile="$(RoslynTargetsPath)\Microsoft.Build.Tasks.CodeAnalysis.dll" Condition="'$(MSBuildAssemblyVersion)' != ''" />
```

Comment:
What's the stuff about MSBuildAssemblyVersion?

Reply:
Good question I think that was also copied from https://github.com/Microsoft/msbuild/blob/master/src/Tasks/Microsoft.Common.tasks

cc @rainersigwald

Reply:
Yes, I copied it. I think it's only for backward compat so not necessary for new tasks.